app/components/show/agreement/details_component.html.erb

```diff
@@ -11,7 +11,7 @@
       <tr>
         <th class="col-3" scope="row">Content type</th>
         <td>
-          <%= render Show::ContentTypeComponent.new(document: @solr_document, version_service: @presenter.version_service) %>
+          <%= render Show::ContentTypeComponent.new(document: @solr_document, presenter: @presenter) %>
```

Comment:
Here and lots of places below: Passing in presenter instead of multiple arguments that are provided by the presenter.